### PR TITLE
Try OSX

### PR DESCRIPTION
**Pull request by jschueller**
@conda-forge-admin please rerender

### CODE REVIEW

**Comment by jschueller** [comment]
This is ready @fcollonval @markaren

**Review by fcollonval — COMMENTED** [review]
I don't use this package since a long time. So I let @markaren have a look.

@jschueller if you don't hear from us in a couple of weeks. Don't hesitate to ping me. I'll gladly add you as maintainer.

File: recipe/0001-Fix-pythonfmu-export-module-on-OSX.patch [hunk 1]
```diff
@@ -0,0 +1,33 @@
+From 01495aa763ef6f82636bdb8556eeb910be9a2f77 Mon Sep 17 00:00:00 2001
+From: Julien Schueller <schueller@phimeca.com>
```
[the remark this answers is not in the document]
Comment (jschueller) [reply recipe/0001-Fix-pythonfmu-export-module-on-OSX.patch]: probably not, fixed

**Comment by jschueller** [comment]
@markaren @fcollonval I guess this should be merged too now the upstream patch has been accepted

**Review by fcollonval — APPROVED** [review]
Thanks @jschueller

**Pull request merged**